src/glmfit.jl

```diff
@@ -89,6 +134,23 @@ function wrkresp(r::GlmResp)
     isempty(r.offset) ? tmp : broadcast!(-, tmp, tmp, r.offset)
 end
 
+"""
+    wrkresp!{T<:FPVector}(v::T, r::GlmResp{T})
+
+Overwrite `v` with the working response of `r`
+"""
+function wrkresp{T<:FPVector}(v::T, r::GlmResp{T})
+    broadcast!(+, v, r.eta, r.wrkresid)
+    isempty(r.offset) ? v : broadcast!(-, v, v, r.offset)
+end
+
+"""
+    wrkwt(r::GlmResp)
+
+Return the working weights.
+
+The result is `r.wrkwt`, not a copy of `r.wrkwt`
```

Comment:
To me, accessing a member of a type directly is poor form.  It is better to use an extractor function if available.  However this one is so trivial that it cold easily be removed.